src/mcts/search.h

```diff
@@ -213,6 +214,25 @@ class SearchWorker {
         moves_left_support_(search_->network_->GetCapabilities().moves_left !=
                             pblczero::NetworkFormat::MOVES_LEFT_NONE) {
     Numa::BindThread(id);
+    for (int i = 0; i < params.GetTaskWorkersPerSearchWorker(); i++) {
```

Comment:
Maybe makes sense not to start threads if MultiGatherEnabled is false?

Reply:
Maybe I should change SearchParams to force TaskWorkersPerSearchWorker to 0 if MultiGather is disabled.